### PR TITLE
empty table fix wokraround

### PR DESCRIPTION
**Pull request by makalaaneesh**

### CODE REVIEW

File: yb-voyager/cmd/exportData.go [hunk 1]
```diff
@@ -276,6 +276,14 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName, ta
 	for _, table := range tableList {
 		dbzmTableList = append(dbzmTableList, table.Qualified.Unquoted)
 	}
+	if exporterRole == SOURCE_DB_EXPORTER_ROLE && changeStreamingIsEnabled(exportType) {
+		err := UpdateMigrationStatusRecord(func(record *MigrationStatusRecord) {
+			record.TableListExportedFromSource = dbzmTableList
+		})
```
Comment (amit-yb) [inline yb-voyager/cmd/exportData.go:282]: Empty tables are not included in the `dbzmTableList` -- they are already filtered by the caller before calling this function.

**Review by amit-yb — COMMENTED** [review]
One important comment.

**Review by amit-yb — APPROVED** [review]
LGTM.

File: yb-voyager/cmd/exportData.go (hunk 1 above)
Comment (makalaaneesh) [reply yb-voyager/cmd/exportData.go:282]: Already handled in a previous PR - wherein empty tables are not filtered out for live migration.

**Pull request merged**